src/dataDisplay/personPanel/personPanelSummary.jsx

```diff
@@ -456,12 +478,12 @@ function PersonPanelSummary(props) {
     const rootClasses = ClassNames(
         classes.root,
         UI_CLASS_NAME,
-        [`${bemClass}`],
+        [`${BEM_CLASS_NAME}`],
         {
-            [classes.expanded]: isExpanded,
+            [classes.isExpanded]: isExpanded,
             [classes.genderFemale]: gender === 'f',
             [classes.genderMale]: gender === 'm',
-            [classes.genderUndefined]: !gender,
+            [classes.genderUndefined]: gender !== 'm' && gender !== 'f',
```

Comment:
Could we convert `gender` to lowercase and then compare like this `gender.toLowerCase() === 'm'`. I believe  it would remove chances of breaking down when capital letters like `M` & `F` are passed as gender

Reply:
Are both capital and lowercase versions "acceptable"?  Perhaps we could do `!includes(['m','f','M','F'], gender)` or something along those lines?

Reply:
My original thought was trying to keep the level of maintenance on this low with only accepting `['f', 'm', null]`, but I like the latter's suggested and will make that change

Reply:
Fixed.

Reply:
Confirmed this looks good in current version.  I'm going to resolve this thread.